ArcGIS-toolbox/ASF_Tools.pyt

```diff
@@ -87,98 +93,100 @@ class UnzipFiles(object):
-            datatype = "GPString",
-            parameterType = "Required",
-            direction = "Input")
-        
+            name="p_outscale",
+            displayName="Scale of output raster (power, amplitude or dB)",
+            datatype="GPString",
+            parameterType="Required",
+            direction="Input")
+
         p_outscale.filter.type = "ValueList"
         p_outscale.filter.list = ["Power", "Amplitude", "dB"]
 
         # Fourth parameter: target directory for output raster
         p_outdir = arcpy.Parameter(
-            name = "p_outdir",
-            displayName = "Destination folder for output raster",
-            datatype = "DEWorkspace",
-            parameterType = "Required",
-            direction = "Input")
+            name="p_outdir",
+            displayName="Destination folder for output raster",
+            datatype="DEWorkspace",
+            parameterType="Required",
+            direction="Input")
 
         # Fifth parameter: file name for output raster
-        p_outname  = arcpy.Parameter(
-            name = "p_outname",
-            displayName = "File name for output raster (must include valid raster file extension)",
-            datatype = "GPString",
-            parameterType = "Required",
-            direction = "Input")
+        p_outname = arcpy.Parameter(
+            name="p_outname",
+            displayName="File name for output raster (must include valid raster file extension)",
+            datatype="GPString",
+            parameterType="Required",
+            direction="Input")
 
         # Sixth parameter: select if output is added to the map
         outYN = arcpy.Parameter(
-            name = "outYN",
-            displayName = "Add output to map",
-            datatype = "GPBoolean",
-            parameterType = "Required",
-            direction = "Input")
+            name="outYN",
+            displayName="Add output to map",
+            datatype="GPBoolean",
+            parameterType="Required",
+            direction="Input")
 
         outYN.value = "true"
 
         # Seventh parameter: output layer to add to project
         outlayer = arcpy.Parameter(
-            name = "outlayer",
-            displayName = "Derived output for final product raster",
-            datatype = "GPRasterLayer",
-            parameterType = "Derived",
-            direction = "Output")
-                        
+            name="outlayer",
+            displayName="Derived output for final product raster",
+            datatype="GPRasterLayer",
+            parameterType="Derived",
+            direction="Output")
+
         params = [p_inpath, p_inscale, p_outscale, p_outdir, p_outname, outYN, outlayer]
         return params
 
     def rasConvert(self):
         """Conversion function"""
-        
+
         if self.inscale == 'Power':
             if self.outscale == 'Amplitude':
                 outSqRt = arcpy.sa.SquareRoot(self.inpath)
```

Comment:
This section is one place where I used pseudo-camelCase. These variable names could be changed if desired.